decart_sdk/models.py

```diff
@@ -0,0 +1,198 @@
+from typing import Literal
+from pydantic import BaseModel, Field, ConfigDict
+from .errors import create_model_not_found_error
+from .types import FileInput
+
+
+RealTimeModels = Literal["mirage", "lucy_v2v_720p_rt"]
+VideoModels = Literal[
+    "lucy-dev-i2v",
+    "lucy-dev-v2v",
+    "lucy-pro-t2v",
+    "lucy-pro-i2v",
+    "lucy-pro-v2v",
+    "lucy-pro-flf2v",
+]
+ImageModels = Literal["lucy-pro-t2i", "lucy-pro-i2i"]
+Model = Literal[RealTimeModels, VideoModels, ImageModels]
+
+
+class ModelDefinition(BaseModel):
+    model_config = ConfigDict(arbitrary_types_allowed=True)
```

Comment:
Could have it in a base class (`class DecartBaseModel` or something) to avoid repeating that